dev_scripts_helpers/update_devops_packages/notebooks/Master_buildmeister_dashboard.ipynb

```diff
@@ -299,6 +299,23 @@
    "source": [
     "http://172.30.2.44/html_coverage/runner_master/"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "027d1b3d",
+   "metadata": {},
+   "source": [
+    "# Code Coverage Page - CodeCov"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6c9219e1",
+   "metadata": {},
+   "source": [
+    "- Helpers: https://app.codecov.io/gh/causify-ai/helpers\n",
+    "- Other Repos: https://app.codecov.io/gh/causify-ai/<repo_name>"
```

Comment:
Can you remove it for now, since it's not implemented in the other repositories.